_data/authors.yml

```diff
@@ -155,3 +155,14 @@ siddharth_sable:
       twitter: "sidsabale" 
       instagram: "siddo.exe"
       youtube: "channel/UCf-RytOIE5VugpaPDZdhobA" 
+      
+Benny: 
+  name: "Benny Ifeanyi Iheagwara"
+  role: "Data analyst, technical writer, and speaker" 
+  avatar: "https://raw.githubusercontent.com/Bennykillua/Project/main/Random/Picone.jpg"
+  bio: " Data analyst • Technical writer • Community manager • OSS contributor • Blockchain Enthusiast."
+  links:
+      github: "Bennnykillua"
```

Comment:
Ah just noticed that you used `Bennnykillua` when your GitHub account is `Bennykillua` (notice the extra `n`).

Reply:
oh sorry that was a typo

Reply:
No problem! Make the commit in this branch itself, so you don't need to make new PR again :)